src/homes/searchSummary.ts

```diff
@@ -0,0 +1,59 @@
+import { camelizeKeys } from "humps"
+import { authenticatedRequest } from "../requests"
+import { token } from "../token"
+import queryBuilder from "./queryBuilder"
+
+import {
+  ICredentials,
+  IEnvironment,
+  IFetchExecutor,
+  IMHBOListing,
+  ISearchParams,
+  IUnparsedMHBOListing
+} from "../types"
+
+/**
+ * Performs a search for mobile homes.
+ *
+ * @param creds The authentication credential for the API request.
+ * @param environment An optional override of the environment to utilize.
+ * @param fetchExecutor An instance of the request executor.
+ * @returns An array of mobile home results.
+ */
+async function searchSummary(
+  params: ISearchParams,
+  creds: ICredentials,
+  environment?: IEnvironment,
+  fetchExecutor?: IFetchExecutor
+): Promise<IMHBOListing[]> {
+  params.detailLevel = "SUMMARY"
```

Comment:
See comment above for searchSummary in communities. Would be nice to reuse the existing search code.